sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -85,6 +85,12 @@ public void describe(SensorDescriptor sensorDescriptor) {
   public void execute(SensorContext sensorContext) {
     File deployDestination = sensorContext.fileSystem().workDir();
     File typescriptLocation = getTypescriptLocation(sensorContext.fileSystem().baseDir());
+    if (typescriptLocation != null) {
+      LOG.debug("TypeScript compiler is found in this directory " + typescriptLocation.getAbsolutePath());
+      LOG.debug("It will be used for analysis of typescript files");
+    } else {
+      LOG.warn("No TypeScript compiler found in your project, global one will be used");
```

Comment:
Here analysis should stop and the message be something like `No TypeScript compiler found in your project, analysis will abort`